### DISCUSSION TITLE
Regional Executors

### DISCUSSION THREAD

**Discussion by kjaymiller** [BlackPythonDevs/blackpythondevs#137] [Ideas]
#### TLDR:

Change the leadership model to have Regional Executors and Vice Regional Executors with a one-year term limit (max 3 years consecutive) to serve under the Black Python Devs Executor Role.

The idea is that a Vice Regional Executor (unless they opt-out) would move into the Regional Executor role and appoint a new Vice Regional Executor creating constant continuity and a leadership cycle of two years with 

#### Goals

- Create a consistent leadership model with opportunities for individuals to get leadership experience
- Create off-ramps for existing leaders to pursue other opportunities
- Reduce the amount of work that is put on the Executor and force delegation of tasks that are better managed by those with a better understanding of the culture.

#### Regions

- Africa
- LatAm
- US/Canada
- Europe

#### Responsibilities

- Aiding in creating BPD user groups in areas lacking meetup support
- Identifying User Groups and Regional Conferences to sponsor
- Sourcing sponsors 

#### In the Event of no Regional Executor

If the Regional Executor steps down. the Vice Regional Executor will be appointed to the Regional Executor.
If there is no Regional Executor or Vice Regional Executor, the Black Python Devs Executora will manage the responsibilities of the Regional Executor and work to find a suitable re

#### Initial Regional Executors

- Africa - Abigail "Afi" Gbadago
- US/Canada - Lazuich Ford
- LatAm - TBD
- Europe - TBD

Trialing this in Africa and hopefully the US as they are our biggest regions.
